cmd/node/node.go

```diff
@@ -526,11 +526,14 @@ func (app *App) initServices(
 	if trtlCfg.BadBeaconVoteDelayLayers == 0 {
 		trtlCfg.BadBeaconVoteDelayLayers = app.Config.LayersPerEpoch
 	}
-	trtl := tortoise.New(app.cachedDB, beaconProtocol,
+	trtl, err := tortoise.New(app.cachedDB, beaconProtocol,
```

Comment:
do you have an idea of how long this would take for the max sliding window size?

Reply:
i don't recall, but the duration is the same for previous solution and this one